### PR TITLE
[4.0] Action log date

### PR DESCRIPTION
**Pull request by brianteeman**
This PR makes the data column here display in the same way as in Privacy Requests.

image

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on de2b89e27edd4ae69b242b72eddc0193e194d639<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25806">issues.joomla.org/tracker/joomla-cms/25806</a>.</sub>

File: administrator/components/com_actionlogs/tmpl/actionlogs/default.php [hunk 1]
```diff
@@ -108,10 +108,8 @@
...
 							</td>
 							<td>
-								<span title="<?php echo HTMLHelper::_('date', $item->log_date, Text::_('DATE_FORMAT_LC6')); ?>">
-									<?php echo HTMLHelper::_('date.relative', $item->log_date); ?>
-								</span>
-								<div role="tooltip" id="<?php echo $item->id; ?>">
+								<?php echo HTMLHelper::_('date.relative', $item->log_date); ?>
+								<div class="small">
 									<?php echo HTMLHelper::_('date', $item->log_date, Text::_('DATE_FORMAT_LC6')); ?>
 								</div>
```
Comment (wilsonge) [inline administrator/components/com_actionlogs/tmpl/actionlogs/default.php:114]: Need to remove the close div here from the tooltip div you've removed

Comment (Quy) [reply administrator/components/com_actionlogs/tmpl/actionlogs/default.php:114]: `<div class="small">` replaces `<div role="tooltip" id="<?php echo $item->id; ?>">`.

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**